snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -5,6 +5,13 @@
 
 init_config:
 
+  ## @param loader - string - optional - default: python
+  ## Check loader to use for all instances. Available loaders:
+  ## - core: (recommended) will use new corecheck SNMP integration
+  ## - python: will use legacy python SNMP integration
+  #
+  loader: core
```

Comment:
Note for reviewer: This is uncommented to encourage new users to use snmp corecheck implementation.

Reply:
what happens if you have this `loader: core` and an instance with `loader: python` ?

Reply:
@pducolin Instance `loader` config will have precedence.